### PR TITLE
An easier setup like starship

### PR DESCRIPTION
**Pull request by ssiyad**
- [x] Setting up is now easy as `eval "$(shelby init bash)"`.

- [x] Can work standalone using `shelby info` command

Tested in bash, not zsh

### CODE REVIEW

**Comment by ssiyad** [comment]
Solves rest of #20 I guess

**Comment by athul** [comment]
Thanks Bruh


Seems like zsh isn't working. Lemme try

**Comment by ssiyad** [comment]
@athul Fixed zsh

**Comment by athul** [comment]
Lol, I had made a PR to fix the zsh stuff

**Comment by athul** [comment]
https://github.com/ssiyad/shelby/pull/1 This one

File: mods/shell.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package mods
+
+var zsh = `autoload -Uz add-zsh-hook
+prompt_shelby_cmd() { dir=$(/usr/local/bin/shelby info) }
+add-zsh-hook precmd prompt_shelby_cmd
+PROMPT=$'$dir\n%(?.%{\e[92m%}.%{\e[91m%})❯%f'`
```
Comment (athul) [inline mods/shell.go]:
```suggestion
var zsh = `
setopt PROMPT_SUBST
NW=$'\n'
PROMPT='$(/usr/local/bin/shelby info)${NW}%(?.%F{green}.%F{red})❯%f'`
```

Comment (athul) [reply mods/shell.go]: Lesser code execution,better loading time

**Comment by ssiyad** [comment]
OOH! Didn't notice that. Is there any functional difference between the two? If yes, we could choose the better one

**Pull request merged**